libvast/builtins/commands/rebuild.cpp

```diff
@@ -460,7 +460,8 @@ struct rebuilder_state {
               });
     const auto num_partitions = current_run_partitions.size();
     self
-      ->request(index, caf::infinite, atom::apply_v, pipeline{std::move(ops)},
+      ->request(index, caf::infinite, atom::apply_v,
+                pipeline{std::move(ops), "<rebuild>"},
```

Comment:
Yes, although in this particular case the roundtripping has always failed since the rebatch operator is not available as an operator plugin of its own. So the roundtripping for this has always failed.

I think we should make rebatch generally available as an operator, and then we wouldn't have this problem anymore.

Reply:
Good point. Perhaps we also need some kind of separation between operators that are user-accessible and those that are not. For instance, the documentation of `rebatch_operator` says: `This operator is intentionally not exposed to the user, as that allows for it to make stricter assumptions about its input`.